litert/vendors/qualcomm/core/builders/broadcast_to_op_builder.cc

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Qualcomm Innovation Center, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "litert/vendors/qualcomm/core/builders/broadcast_to_op_builder.h"
+
+#include <cstdint>
+#include <numeric>
+#include <vector>
+
+#include "litert/vendors/qualcomm/core/builders/op_builder.h"
+#include "litert/vendors/qualcomm/core/tensor_pool.h"
+#include "litert/vendors/qualcomm/core/wrappers/op_wrapper.h"
+#include "litert/vendors/qualcomm/core/wrappers/tensor_wrapper.h"
+#include "third_party/qairt/latest/include/QNN/QnnOpDef.h"
+
+namespace qnn {
+std::vector<std::uint32_t> GetStaticTensorDimention(
+    const std::vector<std::uint32_t>& output,
+    const std::vector<std::uint32_t>& input) {
+  std::vector<std::uint32_t> final_dims(output.size());
+  for (size_t i = 0; i < output.size(); ++i) {
+    final_dims[i] = (output[i] > input[i]) ? output[i] : 1;
+  }
+  return final_dims;
+}
+
+template <typename T>
+TensorWrapper& CreateStaticTensor(TensorPool& tensor_pool,
+                                  const Qnn_DataType_t data_type,
+                                  qnn::TensorWrapperRef output,
+                                  qnn::TensorWrapperRef input) {
```

Comment:
You can directly pass reference instead of reference wrapper.

```suggestion
                                  TensorWrapper& output,
                                  TensorWrapper& input) {
```